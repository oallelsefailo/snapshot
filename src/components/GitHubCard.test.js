import React from "react";
import renderer from "react-test-renderer";
import GitHubCard from "./GitHubCard"; 

test("GitHubCard snapshot", () => {
  const tree = renderer.create(<GitHubCard />).toJSON();
  expect(tree).toMatchSnapshot();
});
