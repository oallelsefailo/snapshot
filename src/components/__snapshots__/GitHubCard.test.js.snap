// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubCard snapshot 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="./images/pic.jpg"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      BRANDON!
    </div>
    <p
      className="card-text"
    >
      INFORMATION NO ONE NEEDS TO KNOW!!
    </p>
    <button
      className="btn btn-primary"
      disabled={false}
      type="button"
    >
      Go somewhere
    </button>
  </div>
</div>
`;
